components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/business-controllers/device.js

```diff
@@ -261,7 +263,9 @@ deviceModule = function () {
                 url, function (responsePayload) {
                     if(!responsePayload["responseText"]){
```

Comment:
Fix the indentation issue.

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/pull/1275